cpp/modmesh/toggle/RadixTree.hpp

```diff
@@ -291,6 +297,7 @@ class CallProfiler
 
 private:
     void print_profiling_result(const RadixTreeNode<CallerProfile> & node, const int depth, std::ostream & outstream) const;
+    static void print_statistics(const RadixTreeNode<CallerProfile> & node, std::ostream & outstream);
```

Comment:
Output arguments are hard to maintain.  Please avoid output arguments.

Reply:
@yungyuc  for `std::ostream`, isn't it a common pattern to use as an output argument?
We can return a string directly, but I think using a stream is more efficient. In addition, we have flexibility to put `std::cout` or `std::stringstream` here.

Reply:
I am not sure if you notice it or not, we also have `void print_profiling_result(const RadixTreeNode<CallerProfile> & node, const int depth, std::ostream & outstream) const;` already.

Reply:
> @yungyuc for `std::ostream`, isn't it a common pattern to use as an output argument? We can return a string directly, but I think using a stream is more efficient. In addition, we have flexibility to put `std::cout` or `std::stringstream` here.

It is common to use ostream as an output argument, but it is better to do it in an `operator>>()` overload.

I can tolerate output arguments when they are with a private function.  It turned out that I did not notice that the helpers are private member functions.  Then I am fine with it.  But not because `std::ostream`.